setup.cfg

```diff
@@ -14,7 +14,7 @@ warn_redundant_casts = True
 warn_unused_configs = True
 check_untyped_defs = True
 disallow_untyped_defs = True
-[mypy-tests/async.*]
```

Comment:
why this? The reason for that was that our sync tests are now 100% typed to ensure that our typings are good.

Reply:
That's because of event now emit the objects, feel free to fix the tests mypy